Foundation/NSHTTPCookieStorage.swift

```diff
@@ -38,7 +38,7 @@ open class HTTPCookieStorage: NSObject {
 
     private static var sharedStorage: HTTPCookieStorage?
     private static var sharedCookieStorages: [String: HTTPCookieStorage] = [:] //for group storage containers
-
+    private var bundleName: String!
```

Comment:
Why store this value? Seems like a good candidate for a computed property (and you get to avoid the implicitly-unwrapped optional if you do that).